packages/cli/src/commands/releasegold/refund-and-finalize.test.ts

```diff
@@ -24,13 +24,18 @@ testWithGanache('releasegold:refund-and-finalize cmd', (web3: Web3) => {
   })
 
   test('can refund gold', async () => {
+    const accounts = await web3.eth.getAccounts()
     await testLocally(Revoke, ['--contract', contractAddress, '--yesreally'])
     const releaseGoldWrapper = new ReleaseGoldWrapper(
       kit.connection,
       newReleaseGold(web3, contractAddress),
       kit.contracts
     )
     const refundAddress = await releaseGoldWrapper.getRefundAddress()
+    // send extra gold to the ReleaseGold contract
```

Comment:
My understanding of this test is that it checks if unreleased gold can be returned to the refundAddress after revoke.

I took a look at the setup for this test, and based on the `releaseGoldExampleConfigs.json`, it seems that the reason this test started failing is because we have reached a 3 year mark after the initial mainnet launch. This caused the full amount to be unlocked and the test to fail, as there was no unreleased gold balance to send back to the refund address.

With the current fix, the test passes as an extra amount is added to the contract, but the initial intended check is bypassed.

An alternate solution would be to modify the `releaseGoldExampleConfigs.json` such that the releaseGold contract still has unreleased gold to refund.

Reply:
Great catch - releaseGoldExampleConfigs.json updated